in_toto/gpg/formats.py

```diff
@@ -58,6 +58,9 @@
 import securesystemslib.formats as ssl_formats
 
 
+GPG_HASH_ALGORITHM_STRING = "pgp+SHA2"
+GPG_PUBKEY_METHOD_STRING = "pgp+rsa-pkcsv1.5"
```

Comment:
Mind also using this constant [here in `gpg/constants.py`](https://github.com/SantiagoTorres/in-toto/blob/f3fcabb4c96f1fceaa41e93b5690e86c12746978/in_toto/gpg/constants.py#L38)?

Reply:
Since you're at it, maybe you can also create a constant for `pgp+dsa-fips-180-2` and use it below in `DSA_PUBKEY_SCHEMA` and as well [here in `gpg/constants.py`](https://github.com/SantiagoTorres/in-toto/blob/f3fcabb4c96f1fceaa41e93b5690e86c12746978/in_toto/gpg/constants.py#L43).

You'd probably have to use a different name for both then, e.g. `RSA_SIGNATURE_METHOD_STRING` and `DSA_SIGNATURE_METHOD_STRING`, what do you think?

Reply:
That may probably be a good idea. As for putting them on `constants.py` it is not trivial, as far as I understand that'd create a circular dependency...

Reply:
Importing and assigning the *constants* defined here in `gpg/constants.py` should not create a circular dependency.

It might look a little weird, but I think that's better than hardcoding the same string twice.